includes/class-edd-db-customers.php

```diff
@@ -424,18 +424,20 @@ public function get_customers( $args = array() ) {
 		if( ! empty( $args['email'] ) ) {
 
 			if( is_array( $args['email'] ) ) {
-				$emails = "'" . implode( "', '", $args['email'] ) . "'";
+
+				$emails_count = count( $args['email'] );
+				$emails_placeholder = array_fill( 0, $emails_count, '%s' );
+				$emails = implode( ', ',  $emails_placeholder ) ;
+				$where .= $wpdb->prepare( " AND `email` IN( $emails ) ",  $args['email'] );
```

Comment:
Ignore previous comment, misread the argument order.

I'd say just formatting updates and this should be good:
```
$emails_count           = count( $args['email'] );
$emails_placeholder = array_fill( 0, $emails_count, '%s' );
$emails                      = implode( ', ',  $emails_placeholder ) ;
$where                      .= $wpdb->prepare( " AND `email` IN( $emails ) ",  $args['email'] );
```